benchexec/tablegenerator/__init__.py

```diff
@@ -60,7 +64,7 @@
 
 DEFAULT_OUTPUT_PATH = "results"
 
-TEMPLATE_FORMATS = ["html", "csv"]
+TEMPLATE_FORMATS = ["html", "csv", "tex"]
```

Comment:
With this change, I think table-generator automatically produces all three formats, does it?

I am not sure whether the tex output is useful enough to have it as default and would leave it disabled. So we can have another list like `DEFAULT_TEMPLATE_FORMATS` and use it as default in `write_table`.

Reply:
I want it such that `table-generator` does not produce the TeX output by default, only if you request it with `-f tex`. But the tests should create all formats and we want to check the expected results for all formats.